pkg/reconciler/taskrun/controller.go

```diff
@@ -67,6 +67,12 @@ func NewController(images pipeline.Images) func(context.Context, configmap.Watch
 			Logger:            logger,
 		}
 
+		// Set up entrypoint cache.
```

Comment:
v minor nit: the code below this comment is reasonably self-explanatory and probably doesn't need the comment